nylas/client/outbox_models.py

```diff
@@ -0,0 +1,179 @@
+from datetime import datetime
+
+from nylas.client.restful_models import RestfulModel, Draft
+from nylas.utils import timestamp_from_dt
+
+
+class OutboxMessage(RestfulModel):
+    attrs = Draft.attrs + [
+        "send_at",
+        "retry_limit_datetime",
+        "original_send_at",
+    ]
+    datetime_attrs = {
+        "send_at": "send_at",
+        "retry_limit_datetime": "retry_limit_datetime",
+        "original_send_at": "original_send_at",
+    }
+    read_only_attrs = {"send_at", "retry_limit_datetime", "original_send_at"}
+    collection_name = "v2/outbox"
+
+    def __init__(self, api):
+        RestfulModel.__init__(self, OutboxMessage, api)
+
+
+class OutboxJobStatus(RestfulModel):
+    attrs = [
+        "account_id",
+        "job_status_id",
+        "status",
+        "original_data",
+    ]
+    collection_name = "v2/outbox"
+
+    def __init__(self, api):
+        RestfulModel.__init__(self, OutboxJobStatus, api)
+
+
+class SendGridVerifiedStatus(RestfulModel):
+    attrs = [
+        "domain_verified",
+        "sender_verified",
+    ]
+    collection_name = "v2/outbox/onboard"
+
+    def __init__(self, api):
+        RestfulModel.__init__(self, SendGridVerifiedStatus, api)
+
+
+class Outbox:
+    def __init__(self, api):
+        self.api = api
+
+    def send(self, draft, send_at, retry_limit_datetime=None):
+        """
+        Send a message via Outbox
+
+        Args:
+            draft (Draft | OutboxMessage): The message to send
+            send_at (datetime | int): The date and time to send the message. If set to 0, Outbox will send this message immediately.
+            retry_limit_datetime (datetime | int): Optional date and time to stop retry attempts for a message.
+
+        Returns:
+            OutboxJobStatus: The Outbox message job status
+
+        Raises:
+            ValueError: If the date and times provided are not valid
+        """
+        json = draft.as_json()
```

Comment:
`json` is a package name, consider usign `draft_json` to avoid confusion.

Reply:
Good call out, done ✅